Tests/MongoSwiftSyncTests/UnifiedTestRunner/SpecialTestOperations.swift

```diff
@@ -0,0 +1,40 @@
+import MongoSwiftSync
+
+struct UnifiedFailPoint: UnifiedOperationProtocol {
+    /// The configureFailpoint command to be executed.
+    let failPoint: BSONDocument
+
+    /// The client entity to use for setting the failpoint.
+    let client: String
+
+    static var knownArguments: Set<String> {
+        ["failPoint", "client"]
+    }
+}
+
+struct AssertSessionNotDirty: UnifiedOperationProtocol {
+    /// The session entity to perform the assertion on.
+    let session: String
+
+    static var knownArguments: Set<String> {
+        ["session"]
+    }
+}
+
+struct AssertDifferentLsidOnLastTwoCommands: UnifiedOperationProtocol {
+    /// Identifier for the client to perform the assertion on.
+    let client: String
+
+    static var knownArguments: Set<String> {
+        Set(["client"])
```

Comment:
is there any reason these ones use the `Set` initializer and the other ones use the literal?

Reply:
good catch, no, I think I just realized partway through that I could use a literal and forgot to fix all of them